### ISSUE TITLE
Accessibility Initial Review / Home+Nav

### ISSUE THREAD

**Issue by o-hook** [porsche-design-system/porsche-design-system#709]
**Scope**  
Review of keyboard accessibility support (and some basic semantics) of Home+Nav product.

**Out of Scope**  
Deep accessibility testing

**Notes**  
H+N should be tested in general with a more accurate test

Subtasks
---
- [ ] Search for best practices how to make mega menu accessible
- [ ] Collect and report improvements and issues
